### DISCUSSION TITLE
Debug Agent Not receiving Message Autogen 0.4

### DISCUSSION THREAD

**Discussion by asheeshgarg** [microsoft/autogen#4229] [General]
I have setup a routed agent and runtime. When I am trying to send a message it doesn’t receives it. How to debug these issues.

**Comment by ekzhu** [comment]
Can you turn on logging?  https://microsoft.github.io/autogen/dev/user-guide/core-user-guide/framework/logging.html

Please attach your minimal reproduction.

**Reply by asheeshgarg** [reply]
import logging

from autogen_core.application.logging import TRACE_LOGGER_NAME
logger = logging.getLogger(f"{TRACE_LOGGER_NAME}.my_module")

I have added Trace logger in my notebook I don't see any logs printed. How to enable it

**Comment by asheeshgarg** [comment]
Figure out the issue thanks for the pointers
